### ISSUE TITLE
Freezed default value

### ISSUE THREAD

**Issue by alejandrogiubel** [artflutter/reactive_forms_generator#170]
Using reactive form generator with freezed the default values ​​that are set to freezed don't seem to work with reactive form, any ideas?

**Comment by alejandrogiubel** [comment]
Here https://github.com/alejandrogiubel/reactive_form_freezed

**Comment by just-abdou** [comment]
hey @alejandrogiubel , did you find any solution. Thanks.

**Comment by alejandrogiubel** [comment]
@just-abdou not yet, just waiting for some update.

**Comment by Nolomon** [comment]
This also leads to returning null for non-nullable disabled fields when calling formModel.submit() although the disabled field has been annotated with @Default('').
I'm currently making all fields that might be disabled nullable, not sure if this is the recommended practice.

**Comment by alejandrogiubel** [comment]
Yes, you're right, I hadn't thought about that solution (very obvious btw).  But in the case where you have a class with several parameters, including some required ones, it doesn't make much sense to define the required parameters at that moment when the idea is to init them when the form is valid. I think the most appropriate thing is that those class parameters that have a default value are initialized in the control with that value.
For example it doesn't make much sense to do:

![Captura de pantalla 2024-10-07 a la(s) 11 11 04 a m](https://github.com/user-attachments/assets/87f9be55-294f-4e1d-87fc-b27044a2ac0f)

If the idea is that the price is required but without a default value

**Comment by alejandrogiubel** [comment]
Hi @vasilich6107. I have tested the beta version and the result is very good. Thank you very much for the great work. Currently it is necessary to initialize the model for the form properties to be initialized, but there are cases where you would only want to partially initialize the model and not all the properties. For example
```
@Rf()
@freezed
class Car with _$Car {
  factory Car({
    @RfControl(
      validators: [RequiredValidator()],
    )
    @Default('Tesla')
    String model,
    @RfControl(
      validators: [RequiredValidator()],
    )
    @Default('Green')
    String color,
    @RfControl(
      validators: [RequiredValidator()],
    )
    @Default(2024)
    int year,
    @RfControl(
      validators: [RequiredValidator()],
    )
    required double price,
  }) = _Car;
}
```
For the previous model need to do
```
model: Car(price: 100000),
```
to initialize the model.
Is there a way without having to initialize the model in the UI that automatically initializes only those properties that have default values ​​in my model?

**Comment by alejandrogiubel** [comment]
Ok ok that's great. Thanks @vasilich6107
